moped-editor/src/views/projects/newProjectView/NewProjectMap.js

```diff
@@ -486,9 +488,13 @@ const NewProjectMap = ({
         />
 
         {/* RENDER LAYERS */}
-        {Object.entries(mapConfig.layerConfigs).map(([sourceName, config]) =>
-          // If a config has a url, it is needs state to update selected/unselected layers
-          config.layerUrl ? (
+        {Object.entries(mapConfig.layerConfigs).map(([sourceName, config]) => {
+          if (isSignalComponent && sourceName !== "drawnByUser") {
```

Comment:
And here's where we filter out the CTN clickable map layers. We are forcing the behavior that if a signal asset is selected only the point geometry from Socrata data can be stored.  If the user wants to select a signal asset but use a different geometry, this should be handled upstream by editing the signal record in the Data Tracker.